cdc/sink/mq.go

```diff
@@ -106,6 +106,10 @@ func newMqSink(
 			avroEncoder.SetValueSchemaManager(valueSchemaManager)
 			return avroEncoder
 		}
+	} else if protocol == codec.ProtocolCanal && !config.EnableOldValue {
+		log.Warn("Old value not enabled when using Canal protocol")
```

Comment:
Looks like it should be log.Error.

Reply:
I think it shoule be `FATAL`

Reply:
What's the purpose of using `log.Fatal` here.

When user creates a new changefeed, `newMqSink` will be called in `verifySink`, IMO return error is enough, we don't need to use fatal everywhrere

Reply:
+1 for `log.Error`